openslides/core/static/js/core/base.js

```diff
@@ -1070,7 +1070,7 @@ angular.module('OpenSlidesApp.core', [
              *  - YOffset: move the editor toolbar 40px up
              */
             getOptions: function () {
-                var extraPluginsString = 'colorbutton,find,sourcedialog,justify,showblocks';
+                var extraPluginsString = 'colorbutton,colordialog,find,sourcedialog,justify,showblocks';
```

Comment:
WHat does this plugin do? I do not have any new button.

Reply:
`colordialog` provides more colors with input for specific color values